package.json

```diff
@@ -59,14 +58,22 @@
     "ldapjs": "1.0.1"
   },
   "devDependencies": {
+    "babel-core": "6.21.0",
+    "babel-loader": "6.2.10",
+    "babel-preset-es2015": "6.18.0",
     "chai": "3.5.0",
     "eslint": "3.10.2",
     "font-awesome": "4.7.0",
     "handlebars": "4.0.6",
+    "handlebars-loader": "1.4.0",
```

Comment:
> comes with its opinionated (as in, it differs from the vanilla Handlebars) format

Does it?

I think we benefit a lot more from using a well test covered loader that solves some of the pain points we hit during creation of this PR. What this allows us to do is `import` templates directly, and the returned template is already compiled ready for consumption. Helpers are turned into generic functions which can also be used by other parts of our code without hitting Handlebars at alal.